lib/pf/iptables.pm

```diff
@@ -154,6 +154,18 @@ sub iptables_generate {
     }
     $tags{'netdata'} .= "-A input-management-if --protocol tcp --match tcp --dport 19999 --jump DROP\n";
 
+    # The dynamic range used to access the fingerbank collector that are connected via a remote connector
+    $tags{'pfconnector'} = "";
```

Comment:
i don´t see pfconnector tag in iptables.conf.example

Reply:
Good catch, it's fixed